BLDC.py

```diff
@@ -1,12 +1,14 @@
 import GPIO
 import PWM
+import time
 
 class Motor:
     def __init__(self, pwmPin, directionPin, frequency):
         GPIO.init()
         self.pwmPin = pwmPin
         self.directionPin = directionPin
         self.speed = 0
+        self.delay = 0.2
```

Comment:
it's better to put this as a static variable instead of an instance variable.
```python
class Motor:
    DELAY = 0.2

    def setSpeed(self, speed):
        time.sleep(self.DELAY)
```